package.json

```diff
@@ -10,11 +10,14 @@
   },
   "homepage": "https://github.com/taskcluster/babel-preset-taskcluster#readme",
   "dependencies": {
-    "babel-preset-es2015": "6.6.0",
+    "babel-plugin-source-map-support-for-6": "0.0.5",
     "babel-plugin-syntax-async-functions": "6.8.0",
     "babel-plugin-transform-async-to-generator": "6.8.0",
     "babel-plugin-transform-runtime": "6.8.0",
     "babel-plugin-transform-strict-mode": "6.8.0",
-    "babel-plugin-source-map-support-for-6": "0.0.5"
+    "babel-preset-es2015": "6.6.0"
+  },
+  "peerDependencies": {
+    "source-map-support": "^0.2.10"
```

Comment:
Why `peerDependencies` over just adding to dependencies? If you depend on this module, place in `dependencies`.